### PR TITLE
Add CoC enforcement manual

### PR DESCRIPTION
**Pull request by Thea-Aldrich**
This document is intended to outline the leadership of the Code of Conduct Working Group, the process by which reports will be handled and possible outcomes. Please see Code of Conduct PR #10364 (https://github.com/zephyrproject-rtos/zephyr/pull/10364) and Reporting Policy PR #10650 (https://github.com/zephyrproject-rtos/zephyr/pull/10650).

### CODE REVIEW

File: doc/code_of_conduct_enforcement_manual.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. _Zephyr_Code_of_Conduct_Enforcement_Manual
+
+Code of Conduct Enforcement Manual
+**********************************
+
+This is the enforcement manual followed by the Zephyr Projects’ Code of Conduct Working Group. It's used when we respond to an issue to make sure we're consistent and fair. It should be considered an internal document, but we're publishing it publicly in the interests of transparency.
```
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]: Please keep line length < 80 characters when possible

Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]:
```suggestion
This is the enforcement manual followed by the Zephyr Project's Code of
Conduct Working Group. It's used when we respond to an issue to make
sure we're consistent, fair, and transparent in our procedures.
```

File: doc/code_of_conduct_enforcement_manual.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+The Code of Conduct Working Group
+*********************************
+
+All responses to reports of conduct violations will be managed by a Code of Conduct Working Group ("the CoC WG").
+The Zephyr Project’s Governing Board ("the board") will establish this working group, comprised of at least five members. One member will be designated chair of the group and will be responsible for all reports back to the board. The board will review membership of the CoC WG on an annual basis. The CoC WG will consist of the project leadership (Governing Board Chair, TSC chair, Marketing Committee Chair, Community Representative and Project Director).
```
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]: Keep line length < 80 characters, and remove trailing spaces.

File: doc/code_of_conduct_enforcement_manual.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
+.. _Zephyr_Code_of_Conduct_Enforcement_Manual
```
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]:
```suggestion
.. _Zephyr_Code_of_Conduct_Enforcement_Manual:
```

File: doc/code_of_conduct_enforcement_manual.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+The Code of Conduct Working Group
+*********************************
+
+All responses to reports of conduct violations will be managed by a Code of Conduct Working Group ("the CoC WG").
+The Zephyr Project’s Governing Board ("the board") will establish this working group, comprised of at least five members. One member will be designated chair of the group and will be responsible for all reports back to the board. The board will review membership of the CoC WG on an annual basis. The CoC WG will consist of the project leadership (Governing Board Chair, TSC chair, Marketing Committee Chair, Community Representative and Project Director). 
+
+How the Code of Conduct Working Group Will Respond to Reports
+*************************************************************
+
+When a report is sent to the committee they will immediately reply to the report to confirm receipt. The reporting email address is **report@zephyrproject.org**. This reply must be sent within 24 hours, and the group should strive to respond much quicker than that.
```
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]: report@zephyrproject.org is not a working email address

File: doc/code_of_conduct_enforcement_manual.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+See the reporting guidelines for details of what reports should contain. If a report doesn't contain enough information, the CoC WG will obtain all relevant data before acting. The CoC WG is empowered to act on the Zephyr Project’s behalf in contacting any individuals involved to get a more complete account of events.
+
+The CoC WG will then review the incident and determine, to the best of their ability:
+ * What happened
```
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]: need a blank line before a list, and don't indent the items:

    The CoC WG will then review the incident and determine, to the best of their ability:

    * What happened

File: doc/code_of_conduct_enforcement_manual.rst [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+If the incident involves physical danger, any member of the working group may -- and should -- act unilaterally to protect safety. This can include cooperating with law enforcement and speaking on behalf of the Zephyr Project. In situations where an individual group member acts unilaterally, they must report their actions to the working group for review within 24 hours.
+
+Resolutions
+***********
+
+The working group must agree on a resolution by consensus. If the group cannot reach consensus and deadlocks for over a week, the group will turn the matter over to the Governing Board for resolution.
+
+Possible responses may include:
+ * Taking no further action (if we determine no violation occurred).
```
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]: Same here.  Add a blank line before the list and don't indent.

File: doc/code_of_conduct_enforcement_manual.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Conflicts of Interest
+*********************
+
+In the event of any conflict of interest a working group member must immediately notify the other members, and recuse themselves if necessary.
+
+*Editor's note: Writing this document posed a unique challenge. Most similar guides are written on the assumption of an in-person event. However, the Zephyr community doesn't exist in one place, and most of the time we're spread out across the world and interact online. This makes trying to define and enforce community standards a different type of challenge. This document is adapted from the Django community, who adopted it from the Ada Initiative template and the PyCon 2013 Procedure for Handling Harassment Incidents, but changed to reflect the nature of our community. It is our expectation that this will be a living document and change as we grow to understand how to meet this challenge and best serve our community and ideals.*
```
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]: Not sure we're that unique -- many open source projects have world-wide contributors, no?

File: doc/code_of_conduct_enforcement_manual.rst (hunk 3 above)
Comment (dbkinder) [inline doc/code_of_conduct_enforcement_manual.rst]: All documents need to be included in the site's table of contents or the doc build will complain about orphan docs.

**Review by dbkinder — CHANGES_REQUESTED** [review]
I think it's best for me to do a editor review after the TSC chimes in on the proposed approach.  I've made suggestions to remove doc build and formatting errors.

**Comment by nashif** [comment]
Can this be closed?

**Comment by nashif** [comment]
closing in favour of #10364.

**Pull request closed without merging**